IDE/STM32Cube/wolfssl_example.c

```diff
@@ -1468,25 +1471,278 @@ static void PrintMemStats(void)
 #endif
 }
 
-#if 0
-static void* wolfMallocCb(size_t size)
+#if !defined(WOLFCRYPT_ONLY) && defined(WOLFSSL_TLS13)
```

Comment:
Please add one more build option for disabling these tests... like `NO_TLS_UART_TEST`. In case someone if one tiny target and trying to test something else.

Reply:
Got it!